install/README.md

```diff
@@ -85,8 +85,8 @@ available observability plugins, to minimize the disk space used for install.
 
 - [Knative Install on Docker for Mac](Knative-with-Docker-for-Mac.md)
 - [Knative Install on Minikube](Knative-with-Minikube.md)
-- [Knative Install on Minishift](Knative-with-Minishift.md)
-- [Knative Install on OpenShift](Knative-with-OpenShift.md)
+- [Knative Install on OpenShift (minishift)](Knative-with-Minishift.md)
```

Comment:
Can you please update the title for Knative-with-Minishift.md and Knative-with-OpenShift.md as well? Both of those two files are using `Knative Install on OpenShift` as the title, it is a bit confuse to me. Comments?